notebooks/tfx_pipelines/cicd/solutions/tfx_cicd_vertex.ipynb

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
<p>Nit: Add space between GCPREGION so it reads YOUR GCP REGION</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Done!</p>